CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+## 0.65.0 / 2022-05-04
+
+The main change introduced by this release is the new v1alpha1 `ScrapeConfig` CRD.
+This implements the [proposal](https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/proposals/202212-scrape-config.md)
+documented in [#5279](https://github.com/prometheus-operator/prometheus-operator/pull/5279)
+and provides a Kubernetes native API to create and manage additional scrape configurations.
```

Comment:
adding a paragraph explaining how to enable the new CRD might be useful (similar to what we did in v0.64.0 for the PrometheusAgent CRD).

Reply:
Added, let me know if it looks good!